### ISSUE TITLE
Websocket ping / pong

### ISSUE THREAD

**Issue by brocaar** [lorabasics/basicstation#29]
Are there plans to support Websocket `PING` / `PONG` messages?

Currently it seems that pings are ignored:

https://github.com/lorabasics/basicstation/blob/c29b8502f8c715daecec6666835da6e981dc820a/src/net.c#L574

```c
    case WSHDR_PING:
    case WSHDR_PONG: {
        LOG(MOD_AIO|WARNING, "[%d] Ignoring WS ping/pong message", conn->netctx.fd);
        break;
}
```

It would be great if at least the Basic Station could respond with a `PONG`.

**Comment by beitler** [comment]
The next release will support `PONG`.

**Comment by brocaar** [comment]
Thanks :-)

**Comment by ogimenez-smtc** [comment]
Hi,

Fixed by 817e6e1, release V2.0.4.
I am closing this issue.
